NanoSolveIT D2.2 ID mappings

egonw opened the pull request:
@dietermaier, plz have a look. In the deliverable you will find ERM identifiers for a few materials that do not have a matching NP identifier yet.

egonw committed:
Sorted by NanoMile ID

egonw committed:
Added more mappings

egonw committed:
Correct core types + the missing HA material

dietermaier commented:
Seems to contain import errors. E.g. NP00194 is correctly mapped to ERM00000400 but incorrectly states "silver" as core while D2.2 lists Ce0.75 Zr0.25 O2
P.S. never have actually worked with Git beyond simplest pull requests. Is there a way to display the index in actual html (apart from downloading the whole thing)?

egonw commented:
> Is there a way to display the index in actual html (apart from downloading the whole thing)?

What index do you mean? Like a line number? If so, then you can click the "Files changed" tab of any pull request (here: https://github.com/NanoCommons/specifications/pull/2/files) and hover over the line numbers (right column) and click the "+" sign that appears. Then you can make a comment on a specific line in the patch / pull request.

About the 'silver' core, I thought I updated them all. I'm sorry about that. I will double check it again.

dietermaier commented:
Sorry, missleading question. I meant the commit which I suppose you ask me to review. https://github.com/NanoCommons/specifications/pull/2/commits/9f69e5924c9ad469521db05c08c76a9cb01e157f it is named "nanomile/index" as far as I can see. It is html code but I would prefer to view it as actual html for readability.

egonw commented:
> It is html code but I would prefer to view it as actual html for readability.

Ah, got it! Yeah, the 'main' branch is pushed with GitHub pages online, but the HTML in a branch is not, I'm afraid.

What you can do, is check out the repo, and then open the HTML in your local browser.

egonw committed:
Fixed another core annotation

egonw commented:
> About the 'silver' core, I thought I updated them all. I'm sorry about that. I will double check it again.

I doubled checked everything. The example you gave was already fixed in the third patch. But I did find one remaining mismatch which I fixed in the 4th commit.
